neps/nep-0372.md

```diff
@@ -0,0 +1,79 @@
+---
+NEP: 372
+Title: Runtime Gas Price
+Author: Ben Kurrek <ben.kurrek@near.org>, Jakob Meier <jakob@near.org>
+DiscussionsTo: https://gov.near.org/t/expose-gas-price-and-contract-access-key-info-within-runtime/24788
+Status: Review
+Type: Standards Track
+Category: Contract
+Created: 14-Jul-2022
+---
+
+## Summary
+
+There is currently no way to query for the current Gas price at runtime from within smart contracts.
+
+## Motivation
+
+As the network becomes more popular, Gas prices may differ between blocks. By adding the ability to query for the price of Gas, it opens many possibilities for developers. Some use cases could include a contract that knows how much Gas is attached to the call or how much Gas has been used so far using the current `prepaid_gas` and `used_gas` functions but they want to convert that to a $NEAR value using the price of Gas.
```

Comment:
Please mention explicit use cases. The text here does not provide sufficient justification for why we need to add this host function